pkg/virt-handler/vm_test.go

```diff
@@ -1725,8 +1727,14 @@ var _ = Describe("VirtualMachineInstance", func() {
 				Iface: &v1.Interface{
 					Name: interfaceName,
 				},
-				PodIP: "2.2.2.2",
+				PodIP:  "2.2.2.2",
+				PodIPs: []string{"2.2.2.2"},
+			}
+
+			if !ipv4Only {
```

Comment:
I would pass the `podIPs` as the argument to the describe table.

Reply:
done